### ISSUE TITLE
SmartAgent permission issue in OpenShift 4.7

### ISSUE THREAD

**Issue by sdodsley** [purestorage/pso-csi#153]
Executing the smart-agent script fails the ping command with `permission denied`.

If I log into one of the csi-node pods smart-config container and try the command manually this is the error I get:

```
/ # ping pso-db-1-0.pso-db.pso-csi -w 2
PING pso-db-1-0.pso-db.pso-csi (10.134.2.16): 56 data bytes
ping: permission denied (are you root?)
```
Unknown if this error occurs on earlier versions of OpenShift.

**Comment by dnix101** [comment]
This is probably related to the container running as a non-root user. In Kubernetes a non-root user does not have access to ping, because it requires a port binding that is not allowed.

**Comment by sdodsley** [comment]
Good catch @dnix101. That is the case. I'll raise a PR to fil this.

**Comment by sdodsley** [comment]
Addressed in #154
